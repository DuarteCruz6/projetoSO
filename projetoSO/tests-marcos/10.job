WRITE [(l8,40)(p7,54)(p7,0)]
READ [l8]
WRITE [(v3,7)]
WRITE [(c4,66)]
WRITE [(d2,64)]
READ [v3]
WRITE [(e6,70)(g5,59)]
SHOW
READ [c4]
BACKUP
WRITE [(s5,85)(e2,93)(y7,80)(d1,8)(d9,15)]
READ [p7]
WRITE [(n8,79)(o5,24)(v9,41)(t4,48)(q3,25)]
READ [v9]
WRITE [(j8,3)(c6,61)]
DELETE [d2]
WRITE [(i9,14)(i7,92)(j0,41)]
DELETE [e2,q3]
WRITE [(b9,24)]
DELETE [v9,n8,v3]
WRITE [(y4,76)]
WRITE [(w6,77)(c3,32)]
WRITE [(e9,5)(s9,13)(h1,38)]
WRITE [(o1,45)(g4,4)(v2,11)]
DELETE [v2,i7,g4,d1]
READ [t4]
READ [c6]
WAIT 3000
WRITE [(d3,33)(w9,82)(s1,75)(e8,96)]
READ [s5]
READ [g5]
READ [c3]
DELETE [e8,l8,s9,p7]
DELETE [y7,o5,e6,j0]
READ [w9]
WRITE [(q3,86)(m5,1)(o5,75)]
BACKUP
DELETE [t4,h1]
WRITE [(a4,84)(b1,54)]
WRITE [(t7,71)(k3,89)(z1,36)(n6,61)(d6,0)]
READ [c4]
WRITE [(n9,30)(n7,26)]
WRITE [(u7,95)(x8,86)(t3,16)]
WRITE [(h0,92)(q0,52)]
READ [j8]
SHOW
WRITE [(d8,58)(i1,89)]
WRITE [(z6,52)]
WRITE [(x1,93)(g4,70)]
WRITE [(q9,65)]
WRITE [(l2,84)]
DELETE [i1,z1,g5]
BACKUP
WRITE [(g6,42)(p2,19)(a6,63)(i5,37)]
BACKUP
DELETE [p2,u7,q9]
DELETE [z6,a4,o1,c4,q0]
WRITE [(q0,75)(h5,58)(o8,1)]
WRITE [(w3,98)]
READ [s1]
DELETE [e9,y4,s5,d3]
READ [j8]
WRITE [(v7,11)(c4,17)(j7,63)(f7,30)(l4,81)]
WRITE [(j6,82)]
BACKUP
WAIT 3000
READ [t7]
WRITE [(j3,99)(n5,55)]
WRITE [(l7,97)(t0,93)(t9,36)(g7,8)]
WRITE [(l3,87)(l8,2)]
WRITE [(j1,0)(y1,73)(m0,34)]
WRITE [(x5,88)(m3,35)(q7,99)]
READ [n6]
WAIT 2000
DELETE [g4,w3,y1]
DELETE [x8,g7,n9,w9,h5]
DELETE [q0,t3]
WRITE [(w2,23)(q0,13)(j2,87)]
