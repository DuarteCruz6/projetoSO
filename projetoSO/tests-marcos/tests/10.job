WRITE [(v6,55)(r0,48)]
WRITE [(l2,34)]
WAIT 3000
WAIT 1000
BACKUP
WRITE [(m6,14)(c9,92)(y6,26)(r2,22)(u5,0)]
READ [c9]
DELETE [l2]
READ [m6]
READ [r2]
READ [r2]
WRITE [(h6,16)(q8,17)]
READ [u5]
WRITE [(f5,60)(r8,4)(g3,87)(f3,3)(i5,37)]
WRITE [(z4,22)(m9,0)]
READ [f5]
WRITE [(c5,16)(i4,32)(f0,30)]
WRITE [(l4,89)(o3,32)]
WRITE [(f4,99)(n0,76)(k4,64)]
READ [c9]
WRITE [(b3,42)(b7,21)]
WRITE [(u1,40)(f8,26)(k1,32)(q5,49)]
BACKUP
SHOW
WRITE [(l2,11)(l3,31)(i6,87)(p3,98)(l2,15)]
READ [n0]
READ [f4]
WRITE [(v0,63)(x7,28)(j9,43)]
WRITE [(t7,30)(o0,39)(m1,29)]
DELETE [v6,r0,v0,p3]
WRITE [(a8,84)(c6,67)(t0,7)]
WRITE [(v4,36)(w0,90)]
WRITE [(l0,49)(f9,26)(d9,83)(j6,0)(d1,0)]
WRITE [(w8,65)(v1,23)(i9,76)(a4,35)]
READ [m1]
WRITE [(g1,95)(z6,28)(s3,66)(w6,38)]
DELETE [f0,x7]
WRITE [(n9,91)]
WRITE [(s5,26)(v9,83)(w1,26)]
WRITE [(r9,91)(n3,12)]
WRITE [(e2,47)]
WRITE [(n7,60)(h3,64)(n5,15)(q3,59)]
DELETE [c9,h6]
READ [m1]
WRITE [(d6,62)(y9,61)(t9,52)(a9,39)(n6,100)]
SHOW
BACKUP
READ [n3]
WRITE [(u8,72)]
WRITE [(h1,17)]
WRITE [(r0,76)(p6,53)(a5,26)]
WRITE [(z1,40)(a3,84)(o7,76)]
BACKUP
WRITE [(s8,9)(b4,39)(b2,25)(b1,18)(l5,81)]
WRITE [(c8,52)(w2,36)]
SHOW
WRITE [(e9,25)(n4,78)]
WRITE [(u6,7)(j2,59)(w9,94)(h6,94)]
WRITE [(z9,54)(v8,25)]
READ [n7]
DELETE [e2,g1,i9]
READ [i4]
READ [m6]
WRITE [(s9,22)(t5,51)(b9,66)(h5,71)]
SHOW
WRITE [(m2,61)(k2,24)]
READ [n7]
WRITE [(y0,0)]
DELETE [t5]
WRITE [(d2,15)(o9,6)(k9,40)(x4,49)]
DELETE [z9,b2,s3]
WRITE [(l1,61)]
WRITE [(l6,76)(u2,61)]
WRITE [(q0,94)(y4,37)(q2,84)(g5,76)]
DELETE [r9,a3,z1]
WRITE [(s6,75)(c0,64)(n1,98)(p7,55)(c4,100)]
WRITE [(a0,54)]
READ [l2]
READ [v9]
READ [q0]
WRITE [(j1,33)]
READ [c4]
WRITE [(c7,63)(e4,95)(h9,99)(p1,22)]
WRITE [(p0,68)(i9,89)(g2,92)(z5,81)]
WRITE [(l7,93)(p8,60)(u4,19)(r4,49)]
