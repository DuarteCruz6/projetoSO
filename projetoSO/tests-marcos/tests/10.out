[(c9,92)]
[(m6,14)]
[(r2,22)]
[(r2,22)]
[(u5,0)]
[(f5,60)]
[(c9,92)]
(b7, 21)
(b3, 42)
(c5, 16)
(c9, 92)
(f8, 26)
(f4, 99)
(f0, 30)
(f3, 3)
(f5, 60)
(g3, 87)
(h6, 16)
(i4, 32)
(i5, 37)
(k1, 32)
(k4, 64)
(l4, 89)
(m9, 0)
(m6, 14)
(n0, 76)
(o3, 32)
(q5, 49)
(q8, 17)
(r8, 4)
(r2, 22)
(r0, 48)
(u1, 40)
(u5, 0)
(v6, 55)
(y6, 26)
(z4, 22)
[(n0,76)]
[(f4,99)]
[(m1,29)]
[(m1,29)]
(a9, 39)
(a4, 35)
(a8, 84)
(b7, 21)
(b3, 42)
(c6, 67)
(c5, 16)
(d6, 62)
(d1, 0)
(d9, 83)
(e2, 47)
(f9, 26)
(f8, 26)
(f4, 99)
(f3, 3)
(f5, 60)
(g1, 95)
(g3, 87)
(h3, 64)
(i9, 76)
(i6, 87)
(i4, 32)
(i5, 37)
(j6, 0)
(j9, 43)
(k1, 32)
(k4, 64)
(l0, 49)
(l3, 31)
(l2, 15)
(l4, 89)
(m1, 29)
(m9, 0)
(m6, 14)
(n6, 100)
(n5, 15)
(n7, 60)
(n3, 12)
(n9, 91)
(n0, 76)
(o0, 39)
(o3, 32)
(q3, 59)
(q5, 49)
(q8, 17)
(r9, 91)
(r8, 4)
(r2, 22)
(s5, 26)
(s3, 66)
(t9, 52)
(t0, 7)
(t7, 30)
(u1, 40)
(u5, 0)
(v9, 83)
(v1, 23)
(v4, 36)
(w1, 26)
(w6, 38)
(w8, 65)
(w0, 90)
(y9, 61)
(y6, 26)
(z6, 28)
(z4, 22)
[(n3,12)]
(a3, 84)
(a5, 26)
(a9, 39)
(a4, 35)
(a8, 84)
(b1, 18)
(b2, 25)
(b4, 39)
(b7, 21)
(b3, 42)
(c8, 52)
(c6, 67)
(c5, 16)
(d6, 62)
(d1, 0)
(d9, 83)
(e2, 47)
(f9, 26)
(f8, 26)
(f4, 99)
(f3, 3)
(f5, 60)
(g1, 95)
(g3, 87)
(h1, 17)
(h3, 64)
(i9, 76)
(i6, 87)
(i4, 32)
(i5, 37)
(j6, 0)
(j9, 43)
(k1, 32)
(k4, 64)
(l5, 81)
(l0, 49)
(l3, 31)
(l2, 15)
(l4, 89)
(m1, 29)
(m9, 0)
(m6, 14)
(n6, 100)
(n5, 15)
(n7, 60)
(n3, 12)
(n9, 91)
(n0, 76)
(o7, 76)
(o0, 39)
(o3, 32)
(p6, 53)
(q3, 59)
(q5, 49)
(q8, 17)
(r0, 76)
(r9, 91)
(r8, 4)
(r2, 22)
(s8, 9)
(s5, 26)
(s3, 66)
(t9, 52)
(t0, 7)
(t7, 30)
(u8, 72)
(u1, 40)
(u5, 0)
(v9, 83)
(v1, 23)
(v4, 36)
(w2, 36)
(w1, 26)
(w6, 38)
(w8, 65)
(w0, 90)
(y9, 61)
(y6, 26)
(z1, 40)
(z6, 28)
(z4, 22)
[(n7,60)]
[(i4,32)]
[(m6,14)]
(a3, 84)
(a5, 26)
(a9, 39)
(a4, 35)
(a8, 84)
(b9, 66)
(b1, 18)
(b2, 25)
(b4, 39)
(b7, 21)
(b3, 42)
(c8, 52)
(c6, 67)
(c5, 16)
(d6, 62)
(d1, 0)
(d9, 83)
(e9, 25)
(f9, 26)
(f8, 26)
(f4, 99)
(f3, 3)
(f5, 60)
(g3, 87)
(h5, 71)
(h6, 94)
(h1, 17)
(h3, 64)
(i6, 87)
(i4, 32)
(i5, 37)
(j2, 59)
(j6, 0)
(j9, 43)
(k1, 32)
(k4, 64)
(l5, 81)
(l0, 49)
(l3, 31)
(l2, 15)
(l4, 89)
(m1, 29)
(m9, 0)
(m6, 14)
(n4, 78)
(n6, 100)
(n5, 15)
(n7, 60)
(n3, 12)
(n9, 91)
(n0, 76)
(o7, 76)
(o0, 39)
(o3, 32)
(p6, 53)
(q3, 59)
(q5, 49)
(q8, 17)
(r0, 76)
(r9, 91)
(r8, 4)
(r2, 22)
(s9, 22)
(s8, 9)
(s5, 26)
(s3, 66)
(t5, 51)
(t9, 52)
(t0, 7)
(t7, 30)
(u6, 7)
(u8, 72)
(u1, 40)
(u5, 0)
(v8, 25)
(v9, 83)
(v1, 23)
(v4, 36)
(w9, 94)
(w2, 36)
(w1, 26)
(w6, 38)
(w8, 65)
(w0, 90)
(y9, 61)
(y6, 26)
(z9, 54)
(z1, 40)
(z6, 28)
(z4, 22)
[(n7,60)]
[(l2,15)]
[(v9,83)]
[(q0,94)]
[(c4,100)]
